qutip/tests/test_gates.py

```diff
@@ -70,6 +73,71 @@ def testSwapGate(self):
         psi_res = swap() * swap() * psi_in
         assert_((psi_in - psi_res).norm() < 1e-12)
 
+    def test1QubitGates(self):
+         """
+         gates: X,Y,Z,S,T gates
+         """
+         N = 7
+         for g in [x_gate, y_gate, z_gate, s_gate, t_gate]:
+
+             a, b = np.random.rand(), np.random.rand()
+             psi1 = (a * basis(2, 0) + b * basis(2, 1)).unit()
+             psi2 = g() * psi1
+
+             psi_rand_list = [rand_ket(2) for k in range(N)]
+
+             for m in range(N):
+
+                 psi_in = tensor([psi1 if k == m else psi_rand_list[k]
+                                  for k in range(N)])
+                 psi_out = tensor([psi2 if k == m else psi_rand_list[k]
+                                   for k in range(N)])
+
+                 G = g(N, m)
+                 psi_res = G * psi_in
+
+                 assert_((psi_out - psi_res).norm() < 1e-12)
```

Comment:
Again, I know you're just following what's going on in the rest of the file (and that's typically a very good thing!).  Now that we've moved to using `pytest`, `np.testing.assert_` is unnecessary - you can just use the keyword `assert`.  It used to be used because numpy abstracted their own tests away from the framework that they used to run them (sort of), and then this provided consistency.  Nowadays, `pytest` will do introspection on any variables that appear in a bare `assert` statement and print out their values, so we actually get much better error reporting by just doing `assert`.